templates/bot/csharp/command-and-response/GettingStarted.txt

```diff
@@ -17,8 +17,11 @@ New to Teams app development or Teams Toolkit? Learn more about
 Teams app manifests, deploying to the cloud, and more in the documentation 
 at https://aka.ms/teams-toolkit-vs-docs
 
+Learn more advanced topic like how to customize your command bot code in 
+tutorials at https://aka.ms/command-bot-tutorial
```

Comment:
VS tutorial and VSC tutorial will be the different wiki page, so we need to use different aka link here.